README.md

```diff
@@ -23,3 +23,41 @@ make autorest
 [releases]: https://github.com/vshn/stardog-userrole-operator/releases
 [license]: https://github.com/vshn/stardog-userrole-operator/blob/master/LICENSE
 [dockerhub]: https://hub.docker.com/r/vshn/stardog-userrole-operator
+
+## Local Development
+
+### Prerequisites
+
+- minikube
+- helm
+- kubectl
+- Stardog image registry credentials
+- Stardog license
+
+### Init local dev environment
+
+```
+minikube start
+
+# Create values.yaml (fill in actual image registry credentials)
+<<EOF > values.yaml
```

Comment:
Are you missing a `cat` here? The provided syntax creates an empty `values.yaml` for me (Linux/bash)